src/TodoItem.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import React from 'react';
+import { Todo } from './types/Todo';
+
+interface Props {
+  todo: Todo;
+  onDelete: (id: number) => void;
+  onToggle: (id: number) => void;
+  loadingIds: number[];
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onDelete,
+  onToggle,
+  loadingIds = [],
+}) => {
+  const isLoading = Array.isArray(loadingIds) && loadingIds.includes(todo.id);
+
+  return (
+    <div className={`todo ${todo.completed ? 'completed' : ''}`} data-cy="Todo">
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) librarry for dynamic managing classnames